docs/source/release_notes/release-notes.rst

```diff
@@ -119,6 +119,12 @@ develop
          - Cassandra client input and output transports are now properly closed.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/3892>`__)
 
+    *    - |fixed| |devbreak|
+         - Stop memoizing the `Supplier` of `TimestampService`, as we **must** get a fresh instance on each `Supplier.get()` call to ensure correctness after leadership elections. Without it, there is a possibility of data corruption.
+           Dev break to force `AtlasDbFactory` and `ServiceDiscoveringAtlasSupplier` to return unified interface of `TimestampService` and `TimestampManagementService`.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/3911>`__)
```

Comment:
:+1: Looks good. Couple of suggestions here:

1. We should probably state that there is only a risk of corruption if you're using services with a leader block in a multinode configuration. Services using External TimeLock, Embedded or Leader with 1 Node won't be affected. 
2. You probably intend ` ``X`` ` for monospacing, rather than `` `X` `` which usually is used for references to other parts of the document.
3. Might be worth including the name of the unified interface (`ManagedTimestampService`).

Reply:
good catch re monospacing, will make these changes